### DISCUSSION TITLE
How to add Modal border on header & footer ?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [ant-design/ant-design#43443] [General]
How to add border on header & Footer??
I saw that the footer is a single DIV but the header is separted, wouldn't it be better to group header into a div for a better CSS customization?? @zombieJ

![Screenshot 2023-07-07 alle 15 07 28](https://github.com/ant-design/ant-design/assets/40239902/20403d6a-08fa-4d60-91d4-9ae150e80cbc)

**Comment by Wxh16144** [comment]
You can set the token for the modal component alone, set `wireframe` to true, and it will add split lines to the header and footer by default; here's a demo: https://codesandbox.io/s/ji-ben-antd-5-6-4-forked-llhcxx?file=/demo.tsx
